### DISCUSSION TITLE
NoAuth Permission

### DISCUSSION THREAD

**Discussion by rucciva** [filebrowser/filebrowser#973] [Ideas]
**Is your feature request related to a problem? Please describe.**
When using `--noauth=true`, filebrowser will create admin user with admin priviledge. It would be better if the permission is configurable.

**Describe the solution you'd like**
add config and env such as `--noauth.admin=false --noauth.create=false`

**Describe alternatives you've considered**

**Additional context**
We are currently using filebrowser as read-only file browser. Current workaround require us to build alpine image and run config, as such

```
  filebrowser:
    image: rucciva/filebrowser:v2.1.1-alpine
    restart: unless-stopped
    environment:
      - FB_DATABASE=/tmp/.filebrowser.db
    working_dir: /home/share
    network_mode: host
    command:
      - |-
        set -euo pipefail
        
        rm $$FB_DATABASE
        filebrowser config init
        filebrowser config set \
          --auth.method=noauth \
          --port=10080 \
          --address=0.0.0.0 \
          --root=/home/share \
          --log=stdout
        filebrowser users add admin admin \
          --lockPassword=true \
          --perm.admin=false \
          --perm.delete=false \
          --perm.create=false \
          --perm.execute=false \
          --perm.rename=false \
          --perm.modify=false
        exec filebrowser
```

**Comment by o1egl** [comment]
@rucciva I'm thinking of adding an anonymous user mode. When you enable this mode each unauthorized user will have read-only rights by default. I think it will solve your issue

**Reply by arky** [reply]
@o1egl Wonder if you had a chance to implement an anonymous user mode? I think it will help me with the following setup https://github.com/filebrowser/filebrowser/discussions/2351

**Comment by rucciva** [comment]
That would be great too, thank you very much.

**Comment by djmoran64** [comment]
I would like this feature as well.

**Comment by Fukitsu** [comment]
I'd like this feature too

**Comment by RomainLG** [comment]
@o1egl Hi there, was the discussed feature implemented?
